BoneReconstructionPlanner/BoneReconstructionPlanner.py

```diff
@@ -337,7 +376,7 @@ def onAddCutPlaneButton(self):
       ind = 0
     else:
       ind = int(name.split('_')[1])%8

```

Comment:
Node names can only be used for displaying information to users. We should not use it to determine what the node contains (for example, because node names are not unique within a scene; also because users can too easily change them and thereby breaking things). Instead, you can use order in the folder or order in node references. If the goal here is to just find a unique color then you can either add a helper function that gives you a unique color by comparing existing colors of nodes in the branch and gives a color that is not similar to any of the existing ones; or a much simpler (not as robust, but probably good enough) option is to add a counter to the parameter node that you increment each time you add a plane (and you use the modulo 8 of this counter as color index).